estate/__manifest__.py

```diff
@@ -5,5 +5,6 @@
     'version': '0.1',
     'application': True,
     'installable': True,
-    'depends': ['base']
+    'depends': ['base'],
+    'data': ['security/ir.model.access.csv']
```

Comment:
In chapter 4, you need to edit the depends line, even though you didn't really changed it. Just to add a `,`. I'd recommend to add it directly. And same comment about the list: you could guess there will be other items, better indent it already to reduce next diff
```suggestion
    'data': [
        'security/ir.model.access.csv',
    ],
```